src/widgets/WordListWidget.vue

```diff
@@ -0,0 +1,87 @@
+<template>
```

Comment:
As part of "repackaging" `scaife-skeleton` and `scaife-widgets`, I'd also like to have a discussion about extensibility for the widgets.

This `template` change would likely land back in the reusable widget (when extracted from EH), but there are other times we might want to think about having further slots / portals for site developers to override within a template, or having the ability to have a project instantiation of a widget use `extends` to override methods, etc.